app/controllers/application_controller.rb

```diff
@@ -55,8 +55,23 @@ def record_not_found
            layout: false
   end
 
-  def user_not_authorized
-    flash[:alert] = "Woops, that's not a page!"
+  def user_not_authorized(exception)
```

Comment:
Unused method argument - `exception`. If it's necessary, use `_` or `_exception` as an argument name to indicate that it won't be used. You can also write as `user_not_authorized(*)` if you want the method to accept any arguments but don't care about them.